system/jlib/jfile.cpp

```diff
@@ -4301,6 +4301,11 @@ IFile * createIFile(const char * filename)
 #endif
 }
 
+void touchFile(const char *filename)
+{
+    Owned<IFile> iFile = createIFile(filename);
```

Comment:
Any possibility that this will fail, and if so, should you handle exceptions?

Reply:
I think correct for touchFile to throw an exception, and ok (if unlikely) for createIFile to throw one on its behalf.
but the IFile->open could in theory return null.
I'll add a check for that and throw an exception if it does.

Reply:
@richardkchapman - please see pushed change